api/controllers/v2/github.ts

```diff
@@ -110,15 +112,16 @@ export const TriggerBuild = async (event: APIGatewayEvent): Promise<APIGatewayPr
   async function createAndInsertJob(path?: string) {
     const repoInfo = await docsetsRepository.getRepo(body.repository.name, path);
     const jobPrefix = repoInfo?.prefix ? repoInfo['prefix'][env] : '';
-    const job = await prepGithubPushPayload(body, repoBranchesRepository, jobPrefix, repoInfo);
+    const job = await prepGithubPushPayload(body, repoBranchesRepository, jobPrefix, repoInfo, path);
 
     consoleLogger.info(job.title, 'Creating Job');
     const jobId = await jobRepository.insertJob(job, c.get('jobsQueueUrl'));
     jobRepository.notify(jobId, c.get('jobUpdatesQueueUrl'), JobStatus.inQueue, 0);
     consoleLogger.info(job.title, `Created Job ${jobId}`);
   }
 
-  if (process.env.FEATURE_FLAG_MONOREPO_PATH === 'true' && body.repository.name === MONOREPO_NAME) {
+  if (body.repository.name === MONOREPO_NAME) {
+    // TODO: MONOREPO feature flag needed here
```

Comment:
Ah, I should have added in the PR notes! I still need to add a check for the feature flag in all these conditionals! The reason I haven't is because we never figured out how with the feature branch. But also... I guess it does seem redundant? I guess the only benefit would be that no one could accidentally release a monorepo site.

Reply:
I think the reason why it doesn't work is because of the [line here](https://github.com/mongodb/docs-worker-pool/blob/b57f0ac0af793b14db7bdf20a2496a1d07bb73be/cdk-infra/lib/constructs/api/webhook-env-construct.ts#L53C69-L53C69). If we remove `.stringValue`, then it should be defined. 

But, if you do find it redundant, I am more than fine with removing it entirely.